### PR TITLE
LCMS Hotfix

### PR DESCRIPTION
**Pull request by icaoberg**
These changes should fix the issues reported by @cebriggs7135

* The presence of `(raw OR mzML)` files  is required.
* The presence of `(txt OR csv)`  files is required.

### CODE REVIEW

**Review by jswelling — APPROVED** [review]

**Review by cebriggs7135 — COMMENTED** [review]
Looks good.

File: src/ingest_validation_tools/directory-schemas/lcms.yaml [hunk 1]
```diff
@@ -1,17 +1,9 @@
 -
-  pattern: 'raw_data/*\.raw'
+  pattern: 'raw_data/*\.(raw|mzML)'
```
Comment (mccalluc) [inline src/ingest_validation_tools/directory-schemas/lcms.yaml:2]: `/*` means 0-or-more-"/": Not what you want.

File: src/ingest_validation_tools/directory-schemas/lcms.yaml [hunk 2]
```diff
@@ -1,17 +1,9 @@
...
-  pattern: 'raw_data/*\.mzML'
-  description: 'Raw mass spectrometry data from an assay of LC-MS, MS, LC-MS Bottom-Up, MS Bottom-Up, LC-MS Top-Down, or MS Top-Down that describes an analyte class of protein, metabolites, lipids, peptides, phosphopeptides, or glycans.'
-  required: True
--
-  pattern: 'ID_search_results/*\.txt'
-  description: 'Identification results. Annotated data describing (qualitative or quantitative) the proteins, metabolites, lipids, peptides, phosphopeptides, or glycans identified from the corresponding raw data.'
-  required: True
--
-  pattern: 'ID_search_results/*\.csv'
+  pattern: 'ID_search_results/*\.(txt|csv)'
```
Comment (mccalluc) [inline src/ingest_validation_tools/directory-schemas/lcms.yaml:6]: ditto.

**Review by mccalluc — COMMENTED** [review]
Regex syntax is still wrong. Until you're more confident about the syntax, I'd suggest providing examples: CI will let you know when examples don't match patterns.

**Pull request merged**